quickstarts/gradle/webapp-wildfly/build.gradle

```
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+    id 'war'
+    id 'org.eclipse.jkube.kubernetes' version '1.6.0-SNAPSHOT'
```

If this works with current release:
```suggestion
    id 'org.eclipse.jkube.kubernetes' version '1.6.0'
```

If it doesn't:
```suggestion
    id 'org.eclipse.jkube.kubernetes' version '1.7.0-SNAPSHOT'
```